http/vertx-web-client/src/test/java/io/quarkus/ts/http/vertx/webclient/PureVertxHttpClientTest.java

```diff
@@ -46,4 +49,10 @@ public void quarkusTestFipsVertxHttpClient() throws InterruptedException {
         done.await(TIMEOUT_SEC, TimeUnit.SECONDS);
         assertEquals(0, done.getCount(), String.format("Missing %d events.", EXPECTED_EVENTS - done.getCount()));
     }
+
+    @AfterEach
+    public void cleanUp(){
+            httpClient.close();
```

Comment:
I think it's unlikely to happen unless webclient initialization fails, but please always check whether `httpclient` var is not null before you try to close it. same for vert.x